### DISCUSSION TITLE
Adding bounding box to reverse geocode API calls

### DISCUSSION THREAD

**Discussion by Sunraj751** [komoot/photon#804] [Q&A]
Hi, i wanted to ask if it is at all possible to add bounding box when doing the reverse geocoding calls. 

**Trying to get**

- As mentioned in documentation, we can use reverse geocoding endpoint feature to find pharmacies around.
- Is it possible to do a query where we can "Find pharmacies within this bounding box" (**_basically doping a radius search_**) ?

Thanks!

**Comment by Sunraj751** [comment]
I think I found the answer, thanks to an old chat/comment (https://github.com/komoot/photon/issues/295#issuecomment-358699671), where it is shown that there is a **radius** feature for exactly this purpose. 

An example of doing a radius search (radius is in KMs): 

1. `http://photon.komoot.io/reverse?lat=45.78205&lon=-108.07531&radius=5`

**Reply by mtmail** [reply]
It’s komoot.io, not komoot.de these days
